app/models/gws/staff_record/group.rb

```diff
@@ -0,0 +1,38 @@
+class Gws::StaffRecord::Group
+  include SS::Document
+  include Gws::Referenceable
+  include Gws::Reference::User
+  include Gws::Reference::Site
+  include Gws::StaffRecord::Yearly
+  include Gws::Addon::ReadableSetting
+  include Gws::Addon::GroupPermission
+  include Gws::Addon::History
+  include Gws::Export
+
+  seqid :id
+  field :name, type: String
+  field :order, type: Integer, default: 0
+  field :seating_chart_url, type: String
+
+  permit_params :name, :order, :seating_chart_url
+
+  validates :name, presence: true, uniqueness: { scope: [:site_id, :year] }
+
+  default_scope -> { order_by order: 1 }
```

Comment:
__[rails_best_practices]__
* default_scope is evil [:ref](http://rails-bestpractices.com/posts/2013/06/15/default_scope-is-evil/)

<!-- Comment by [SideCI](https://sideci.com) -->